### ISSUE TITLE
Outputting YAML fails

### ISSUE THREAD

**Issue by bubbleupdotnet** [monken/cfn-include#29]
On every template we have, outputting to YAML fails with the following error:

```
{ [YAMLException: unacceptable kind of an object to dump [object Undefined]]
  name: 'YAMLException',
  reason: 'unacceptable kind of an object to dump [object Undefined]',
  mark: undefined,
  message: 'unacceptable kind of an object to dump [object Undefined]' }
```

These same templates output to JSON without issue. I'm on Linux Mint 18.2, cfn-include 0.10.1.

**Comment by monken** [comment]
Could you please provide a minimal test case of an input template that fails to output to yaml?

**Comment by monken** [comment]
published v0.10.2 to npm which includes the fix.
